sensor/kubernetes/fake/deployment.go

```diff
@@ -268,9 +268,12 @@ func getContainer(workload ContainerWorkload) corev1.Container {
 			},
 			{
 				Name:  "ROX_SECRET_PASSWORD",
-				Value: "roxapitoken",
+				Value: "secretpassword",
 				ValueFrom: &corev1.EnvVarSource{
 					SecretKeyRef: &corev1.SecretKeySelector{
+						LocalObjectReference: corev1.LocalObjectReference{
```

Comment:
This was not a valid deployment in the fake k8s, but fixes the issue where the secret query was empty which was due to an empty name